docs/modules/hdfs/pages/usage-guide/resources.adoc

```diff
@@ -84,7 +91,7 @@ spec:
           limit: 512Mi
         storage:
           data:
-            capacity: 512Mi
+            capacity: 1Gi
```

Comment:
I guess we should be consistent here. Most of the resource related PRs use `Mi`. So in this case `1024MI` and `2048Mi`. Same goes for the `Quantity` in `JournalNodeConfigFragment`.

Reply:
I was aware of this and did it intentionally, as I prefer using Gi over Mi over Ki over writing raw numbers ;)
What will end up in k8s will be 1Gi anyway AFAIK.
But I'm happy to let the majority decide or change it back when you have stronger feelings than I have ;)

Reply:
Yeah i think 1024Gi will show up as 1Gi. I would like to set what will actually be "returned" by the api server for easier testing? Meaning always transform to the highest unit still keeping the natural number.
Its similar for 1000m and "1" CPU.
But yes we should keep this consistent.

Reply:
Let's flip a coin. Head or tail?
Jokes aside, would let you decide

Reply:
For now, I would keep it consistent. Maybe we want to come up with an internal policy (and document it) which syntax / unit we use, and also at what value we switch from one unit to another.

Reply:
Wait, are you talking about the pvc size or Memory? I *think* we use Mi or Memory in most operators and Gi for pvc sizes (as every pvc has at a minimum 1Gi)

Reply:
Added it to Arch meeting

Reply:
Ok, did walk through the code and we actually use `Mi` for all the memory declaration, event when we could have used Gi

Reply:
Okay that's great! Putting further discussion into the arch meeting seems like a good idea.